org.eclipse.wildwebdeveloper.tests/src/org/eclipse/wildwebdeveloper/tests/TestLanguageServers.java

```diff
@@ -175,7 +175,7 @@ protected boolean condition() {
 					return false;
 				}
 			}
-		}.waitForCondition(PlatformUI.getWorkbench().getDisplay(), 3000));
+		}.waitForCondition(PlatformUI.getWorkbench().getDisplay(), 10000));
```

Comment:
The fact that you need to grow the timeout here shows that pulling the catalog should be happening asynchronously IMHO.